src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/AssemblyReferenceResolverTests.cs

```diff
@@ -28,7 +28,8 @@ public void ShouldAnalyzeProjectDependencies()
                 It.IsAny<string>(),
                 It.IsAny<string>(),
                 It.IsAny<string>(),
-                null))
```

Comment:
I think it's time to refactor this test class.

1. Create the subject under test in the constructor of the test class.
2. Create a helper method to arrange the result of the `processExecutorMock`. For example: `ArrangeProcessExecutor(string result)`